src/Tools/ExternalAccess/LiveShare/CodeActions/RoslynCodeActionProvider.cs

```diff
@@ -53,31 +54,39 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                 Range = ProtocolConversions.TextSpanToRange(span, text)
             };
 
-            var commands = await lspClient.RequestAsync(LSP.Methods.TextDocumentCodeAction, codeActionParams, cancellationToken).ConfigureAwait(false);
+            var commands = await lspClient.RequestAsync(LSP.Methods.TextDocumentCodeAction.ToLSRequest(), codeActionParams, cancellationToken).ConfigureAwait(false);
             if (commands == null)
             {
                 return;
             }
 
             foreach (var command in commands)
             {
-                if (command is LSP.Command lspCommand)
+                LSP.Command lspCommand = null;
+                try
+                {
+                    lspCommand = ((JObject)command).ToObject<LSP.Command>();
+                }
+                catch (Exception) { }
```

Comment:
2 parts I don't like here - 
1.  catch all - do we know what exceptions are likely to be thrown?  I'm guessing some kind of JsonSerializationException and possibly invalid cast to JObject.
2.  No body in the catch - I think you could use `continue` here, there's no point in preceding in this iteration if we can't deserialize to a command. I don't know if we should NFW @heejaechang any thoughts on that?

Reply:
1. The cast to JObject will always be valid. I'll explicitly catch a `JsonException`.
2. Sure. Fine with going with `continue`, as it's identical to the current behavior (but more explicit).